central/reports/scheduler/schedule_test.go

```diff
@@ -0,0 +1,94 @@
+package scheduler
+
+import (
+	"testing"
+	"text/template"
+
+	"github.com/stackrox/rox/pkg/branding"
+	"github.com/stackrox/rox/pkg/fixtures"
+	"github.com/stackrox/rox/pkg/templates"
+	"github.com/stackrox/rox/pkg/testutils/envisolator"
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	expectedVulnReportEmailTemplateRhacsBrandingWithPlaceholders = `
+	Red Hat Advanced Cluster Security for Kubernetes has found vulnerabilities associated with the running container images owned by your organization. Please review the attached vulnerability report {{.WhichVulns}} for {{.DateStr}}.
+
+	To address these findings, please review the impacted software packages in the container images running within deployments you are responsible for and update them to a version containing the fix, if one is available.`
+
+	expectedVulnReportEmailTemplateStackroxBrandingWithPlaceholders = `
+	StackRox has found vulnerabilities associated with the running container images owned by your organization. Please review the attached vulnerability report {{.WhichVulns}} for {{.DateStr}}.
+
+	To address these findings, please review the impacted software packages in the container images running within deployments you are responsible for and update them to a version containing the fix, if one is available.`
+
+	expectedNoVulnsFoundEmailTemplateRhacsBranding = `
+	Red Hat Advanced Cluster Security for Kubernetes has found zero vulnerabilities associated with the running container images owned by your organization.`
+
+	expectedNoVulnsFoundEmailTemplateStackroxBranding = `
+	StackRox has found zero vulnerabilities associated with the running container images owned by your organization.`
+
+	timeStr = `today`
+)
+
+type vulnsAndDate struct {
+	WhichVulns string
+	DateStr    string
+}
+
+func generateExpectedVulnReportEmailTemplates(t *testing.T) (string, string) {
+	data := &vulnsAndDate{
+		WhichVulns: "for all vulnerabilities",
+		DateStr:    timeStr,
+	}
+
+	tmpl, err := template.New("VulnsRHACS").Parse(expectedVulnReportEmailTemplateRhacsBrandingWithPlaceholders)
+	assert.NoError(t, err)
+	expectedVulnReportEmailTemplateRhacsBranding, err := templates.ExecuteToString(tmpl, data)
+	assert.NoError(t, err)
+
+	tmpl, err = template.New("VulnsStackrox").Parse(expectedVulnReportEmailTemplateStackroxBrandingWithPlaceholders)
+	assert.NoError(t, err)
+	expectedVulnReportEmailTemplateStackroxBranding, err := templates.ExecuteToString(tmpl, data)
+	assert.NoError(t, err)
+
+	return expectedVulnReportEmailTemplateRhacsBranding, expectedVulnReportEmailTemplateStackroxBranding
+}
+
+func TestFormatVulnMessage(t *testing.T) {
+	envIsolator := envisolator.NewEnvIsolator(t)
```

Comment:
Move this to `SetupTest` hook

Reply:
Done, but used `SetupSuite` instead since initializing the `envIsolator` once should suffice.